src/tmpfiles/tmpfiles.c

```diff
@@ -2306,17 +2346,27 @@ static int clean_item_instance(Item *i, const char* instance) {
                         sx.stx_ino != ps.st_ino;
         }
 
-        log_debug("Cleanup threshold for %s \"%s\" is %s",
-                  mountpoint ? "mount point" : "directory",
-                  instance,
-                  format_timestamp_style(timestamp, sizeof(timestamp), cutoff, TIMESTAMP_US));
+        if (DEBUG_LOGGING) {
+                _cleanup_free_ char *ab_f = NULL, *ab_d = NULL;
+                if (!(ab_f = age_by_to_string(i->age_by_file, false)))
```

Comment:
we usually would write that in two lines, i.e. not pack too much stuff into the same expression, i.e.:

```c
ab_f = …;
if (!ab_f) …
```

Reply:
ACK.